<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>massimportClass</class>
 <widget class="QDialog" name="massimportClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mass Import</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/devices48/resources/tr48x48-0016.png</normaloff>:/devices48/resources/tr48x48-0016.png</iconset>
  </property>
  <widget class="QPushButton" name="newPlaylistButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>New Playlist</string>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/icons/resources/filenew.png</normaloff>:/icons/resources/filenew.png</iconset>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>911</width>
     <height>251</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Track</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Artist - Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Version</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Genre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Length</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dupe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Action</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="recordSource">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>390</y>
     <width>125</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>1024</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="recordSourceId">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>390</y>
     <width>70</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="maxLength">
    <number>1024</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_67">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Recording source:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tags">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>605</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>2048</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_63">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>510</y>
     <width>32</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Tags:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
